internal/pkg/term/selector/selector.go

```diff
@@ -71,7 +71,8 @@ const (
 	taskFinalMsg        = "Task:"
 	workloadFinalMsg    = "Name:"
 	dockerfileFinalMsg  = "Dockerfile:"
-	topicFinalmessage   = "Topic subscriptions:"
+	topicFinalMessage   = "Topic subscriptions:"
```

Comment:
nit: thanks for fixing this, for consistency:
```suggestion
	topicFinalMsg   = "Topic subscriptions:"
```